trustfall_core/src/interpreter/helpers.rs

```diff
@@ -63,6 +66,110 @@ pub fn resolve_coercion_with<'vertex, Vertex: Debug + Clone + 'vertex>(
 }
 
 /// Helper for making property resolver functions based on fields.
+///
+/// Generally used with [`resolve_property_with`].
+///
+/// Retrieves a [FieldValue] from a vertex by converting it to the proper type,
+/// and then retrieving the field of a struct.
+///
+/// If the property is computed by a function, use
+/// [`accessor_property!`](crate::accessor_property) instead.
+///
+/// # Examples
+/// ```
+/// # use std::rc::Rc;
+/// # use trustfall_core::{
+/// #    field_property,
+/// #    ir::FieldValue,
+/// #    interpreter::{
+/// #            helpers::resolve_property_with, basic_adapter::{ContextIterator, ContextOutcomeIterator}
+/// #        }
+/// #    };
```

Comment:
```suggestion
/// # use trustfall_core::{
/// #     field_property,
/// #     interpreter::{
/// #         basic_adapter::{ContextIterator, ContextOutcomeIterator},
/// #         helpers::resolve_property_with,
/// #     },
/// #     ir::FieldValue,
/// # };
```
I just ran this through `cargo fmt`.